pennylane/estimator/resource_operator.py

```diff
@@ -289,6 +304,9 @@ def controlled_resource_decomp(
     ) -> list[GateCount]:
         r"""Returns a list representing the resources for a controlled version of the operator.
 
+        For ResourceOperators that don't define a `controlled_resource_decomp`, this will be the default
+        `controlled_resource_decomp` method.
```

Comment:
Can you add a sentence here saying the default every operator in the decomposition will be controlled.